pandas_gbq/tests/test_gbq.py

```diff
@@ -402,129 +371,61 @@ def test_read_gbq_with_corrupted_private_key_json_should_fail(self):
                 private_key=re.sub('[a-z]', '9', _get_private_key_contents()))
 
 
-class TestReadGBQIntegration(object):
-
-    @classmethod
-    def setup_class(cls):
-        # - GLOBAL CLASS FIXTURES -
-        #   put here any instruction you want to execute only *ONCE* *BEFORE*
-        #   executing *ALL* tests described below.
-
-        _skip_if_no_project_id()
-
-        _setup_common()
-
-    def setup_method(self, method):
-        # - PER-TEST FIXTURES -
-        # put here any instruction you want to be run *BEFORE* *EVERY* test is
-        # executed.
-        pass
-
-    @classmethod
-    def teardown_class(cls):
-        # - GLOBAL CLASS FIXTURES -
-        # put here any instruction you want to execute only *ONCE* *AFTER*
-        # executing all tests.
-        pass
-
-    def teardown_method(self, method):
-        # - PER-TEST FIXTURES -
-        # put here any instructions you want to be run *AFTER* *EVERY* test is
-        # executed.
-        pass
-
-    def test_should_read_as_user_account(self):
-        _skip_local_auth_if_in_travis_env()
+def test_should_read(project, credentials):
```

Comment:
As opposed to my comment below, I do like this test using all three credentials types.

I think it's good to verify that with a speedy query like this one.